.github/workflows/pr-docker.yml

```diff
@@ -0,0 +1,18 @@
+name: PR Docker
```

Comment:
'Ubuntu Docker'? Docker also exists for Windows (both host and container)

Reply:
Do you think I should rename it to `PR Ubuntu Docker` or something?

Reply:
Ok, I'll change